maint/local_python/binding_c.py

```diff
@@ -133,84 +200,173 @@ def process_func_parameters(func):
         if group_count > 0:
             t = ''
             for j in range(group_count):
-                a = func['params'][i + j]['name']
+                pj = func['params'][i + j]
```

Comment:
what is `pj`?

Reply:
A temporary variable name for `p` since `p` is already used.  Will `p_j` reads better?

Reply:
changed into `temp_p`